src/EnergyPlus/Humidifiers.cc

```diff
@@ -862,12 +861,12 @@ namespace Humidifiers {
             }
 
             NomCap = RhoH2O(Constant::InitConvTemp) * NomCapVol;
-            int RefrigerantIndex = FluidProperties::GetRefrigNum(state, format(fluidNameSteam));
-            int WaterIndex = FluidProperties::GetGlycolNum(state, format(fluidNameWater));
-            SteamSatEnthalpy = GetSatEnthalpyRefrig(state, format(fluidNameSteam), TSteam, 1.0, RefrigerantIndex, CalledFrom);
-            WaterSatEnthalpy = GetSatEnthalpyRefrig(state, format(fluidNameSteam), TSteam, 0.0, RefrigerantIndex, CalledFrom);
-            WaterSpecHeatAvg = 0.5 * (GetSpecificHeatGlycol(state, format(fluidNameWater), TSteam, WaterIndex, CalledFrom) +
-                                      GetSpecificHeatGlycol(state, format(fluidNameWater), Tref, WaterIndex, CalledFrom));
+
+            auto *water = FluidProperties::GetWater(state);
```

Comment:
An index-based lookup like `FluidProperties::GetPropertyRefrig(state, refrigNum, x, y, z);` requires three consecutive indirections: i) `state.dataFluidProperties`, ii) `dataFluidProperties->refrigs`, iii) `refrigs(refrigNum)`. Saving the object pointer and making this into a method eliminates these three consecutive lookups. One small step for EnergyPlus.